File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio5 Área de  un rectángulo/archives/5.php
<?php
$num1 = $_POST["num1"];
$num2 = $_POST["num2"];
$area = round($num1 * $num2,2);

$mensaje = "Area = Base * Altura
			<br><br>Base = $num1
			<br><br>Altura =$num2 
			<br><br>Area = $num1 * $num2
			<br><br>Area = $area";

echo $mensaje;
	
	

















/* if ($metodo == "php") { */
	// Sumar utilizando PHP
/* 	$resultado = $num1 * $num2; */
/* 	echo "la multiplicacion de <br> $num1 * $num2 es: ".$resultado;
} elseif ($metodo == "python") { */
	// Sumar utilizando Python
/* 	$output = shell_exec("python 1.py " . $num1 . " " . $num2);
	echo $output;
} else {
	echo "Método no válido";
} */

File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio5 Área de  un rectángulo/archives/5.py
import sys

# Obtener los argumentos de la línea de comandos
num1 = float(sys.argv[1])
num2 = float(sys.argv[2])
area = round(num1*num2, 2)


print(f"""Area = Base * Altura
			<br><br>Base = {num1}
			<br><br>Altura ={num2} 
			<br><br>Area = {num1} * {num2}
			<br><br>Area = {area} """)
